versions.yaml

```diff
@@ -75,7 +75,7 @@ assets:
       url: "https://github.com/cloud-hypervisor/cloud-hypervisor"
       uscan-url: >-
         https://github.com/cloud-hypervisor/cloud-hypervisor/tags.*/v?(\d\S+)\.tar\.gz
-      version: "v29.0"
+      version: "v30.0"
```

Comment:
Hi @GeorginaKin - can we revert this change as I don't think CLH should be updated in `CC`

Reply:
Ah yeah apologies! Totally forgot with all the conflicts going on- thank you!!